doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc

```diff
@@ -9,11 +9,27 @@ image::admin-and-config/task-inbox.png[Task inbox]
 
 Users with the `process-admin` role can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
 
+[NOTE]
+=====
+
+The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions. To display all the tasks in the *Menu* → *Manage* → *Tasks* page, users with `process-admin` role or `admin` role must belong to `Administrators` group.
+
+To assign users to `Administrators` group, open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:
+
+[source]
+----
+<system-properties>
+        ...
+        <property name="org.jbpm.ht.admin.group" value="admin"/>
+    </system-properties>
+----
+=====
+
```

Comment:
Note:
The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
To be able to manage tasks the user needs to be designed as process administrator, asserting one of this conditions: 
- User is the one specified as 'admin user' (by default 'administrator') 
- User belongs to 'admin group'.(by default 'administrators')
Both values can be configurated via system properties. To change the default value open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:  

<system-properties>
    ...
    <property name="org.jbpm.ht.admin.user" value="administrator"/>
    <property name="org.jbpm.ht.admin.group" value="administrators"/>
</system-properties>